swan-lake/integration-tools/health-tool.md

```diff
@@ -32,8 +32,9 @@ health:2.0.0 successfully set as the active version.
 
 The Ballerina Health tool supports two main usages as follows.
```

Comment:
Shall we change this to, "The main uses of the Ballerina Health tool are as follows:"

Reply:
Updated: The Ballerina Health tool has two primary usages.